karma.conf.js

```diff
@@ -24,16 +24,27 @@
 
 const devMode = process.env.NODE_ENV !== 'production';
 const browsers = [process.env.NODE_ENV === 'debug' ? 'ChromeDebugging' : 'ChromeHeadless'];
+const coverageEnabled = process.env.COVERAGE === 'true';
```

Comment:
I think we should just always run coverage when tests are run. No?

Reply:
`npm test` is lot quicker than `npm run test-coverage`, so its nice to have option for quick pass/fail check

Reply:
@nikhilmandlik Fair enough!